am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/FilterResourceIntegrationTest.java

```diff
@@ -329,7 +338,16 @@ void whenExplicitAccessWithDifferentRelationshipParentChildAttributeDiffPermissi
                     JsonPointer.valueOf(CHILD_ATTRIBUTE), ImmutableSet.of(CREATE, READ)))
                 .accessType(EXPLICIT)
                 .build())
-            .relationships(ImmutableSet.of(OTHER_ROLE_NAME, ROLE_NAME))
+            .relationships(ImmutableSet.of(otherRoleName, roleName))
```

Comment:
we have a relationship variable, use or remove

Reply:
👍